integration_test/test_videoroom/test/integration/simulcast_test.exs

```diff
@@ -320,4 +307,17 @@ defmodule TestVideoroom.Integration.SimulcastTest do
       send(pid, acc)
     end
   end
+
+  defp assert_sender_receiver_stats(stage, encoding, sender_stats_samples, receiver_stats_samples) do
```

Comment:
I would consider renaming `encoding` to `receiver_selected_encoding` and `stage` to `tag`

Reply:
Maybe `receiver_encoding`? Anything that will indicate or tell us that this is related to receiver